test/ad/distributions.jl

```diff
@@ -143,7 +143,7 @@
         DistSpec(Levy, (0.0,), 0.5),
         DistSpec(Levy, (0.0, 2.0), 0.5),
 
-        DistSpec((a, b) -> LocationScale(a, b, Normal()), (1.0, 2.0), 0.5),
+        DistSpec((a, b) -> a + b * Beta(), (1.0, 2.0), 2.0),
```

Comment:
Maybe consider adding a comment here to indicate it is meant to test an `AffineDistribution` distribution. The syntax `a+b*Beta()` is quite subtle, I think.

Reply:
```suggestion
        # Test AffineDistribution
        DistSpec((a, b) -> a + b * Beta(), (1.0, 2.0), 2.0),
```